ex02.ex

```diff
@@ -53,9 +53,10 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
-
-
+  first2equal = fn
```

Comment:
I received no failures while running the code. The code is followed in next line.